desktop/sources/index.html

```diff
@@ -6,7 +6,9 @@
     <script type="text/javascript" src="scripts/lib/theme.js"></script>
     <script type="text/javascript" src="scripts/lib/history.js"></script>
     <script type="text/javascript" src="scripts/lib/source.js"></script>
-    <script type="text/javascript" src="scripts/core/library.js"></script>
+    <script type="text/javascript" src="scripts/core/library/library.js"></script>
+    <script type="text/javascript" src="scripts/core/library/base.js"></script>
+    <script type="text/javascript" src="scripts/core/library/default.js"></script>
```

Comment:
One of the uglier parts. Is there a better way to organize and load these so that new definitions can be added without modifying `index.js`? Can one JS file import another?

Ideally this would be user loadable.